SoraFoundation/Classes/Timer/CountdownTimer.swift

```diff
@@ -64,11 +75,12 @@ public final class CountdownTimer: NSObject {
     }
 
     private func scheduleTimer() {
-        timer = Timer.scheduledTimer(timeInterval: 1.0,
```

Comment:
It will not work if notification interval greater than countdown interval. For example, we set notification interval 1 day and countdown interval 1 hour. So we expect that timer completes in 1 hour but in reality it will complete in 1 day.